common/views/components/ModalMoreFilters/ModalMoreFilters.tsx

```diff
@@ -0,0 +1,359 @@
+  subjectsFilter: CatalogueAggregationBucket[];
+} & sharedFiltersProps;
+
+const ModalInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-width: 320px;
+  max-width: 600px;
+  ${props => props.theme.media.large`
+    width: 600px;
+    top: 10px;
+  `}
+  position: relative;
+  top: 15px;
+  overflow-y: auto;
+  max-height: 80vh;
+`;
+
+// shared styles
+const FilterSection = styled(Space).attrs({
+  h: { size: 'l', properties: ['padding-left', 'padding-right'] },
+  v: { size: 'l', properties: ['padding-top', 'padding-bottom'] },
+  className: classNames({
+    'border-bottom-width-1 border-color-pumice': true,
+  }),
+})``;
+
+const FiltersFooter = styled(Space).attrs({
+  h: { size: 'l', properties: ['padding-left', 'padding-right'] },
+  v: { size: 'l', properties: ['padding-top', 'padding-bottom'] },
+  className: classNames({
+    'bg-white border-color-pumice border-top-width-1 flex flex--v-center flex--h-space-between': true,
+  }),
+})`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 60px;
+`;
+
+const FiltersHeader = styled(Space).attrs({
+  h: { size: 'm', properties: ['padding-left', 'padding-right'] },
+  v: { size: 'm', properties: ['padding-top', 'padding-bottom'] },
+  className: classNames({
+    'border-color-pumice border-bottom-width-1 absolute': true,
+  }),
+})`
+  text-align: center;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+`;
+
+const OverrideModalWindow = styled(Space).attrs<BaseModalProps>({
+  v: { size: 'xl', properties: ['padding-top', 'padding-bottom'] },
+  className: classNames({
+    'shadow bg-white font-black': true,
+  }),
+})<BaseModalProps>`
+  z-index: 10001;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  position: fixed;
+  transition: opacity 350ms ease, transform 350ms ease;
+
+  &,
+  &.fade-exit-done {
+    z-index: -1;
+    pointer-events: none;
+  }
+  &.fade-enter,
+  &.fade-exit,
+  &.fade-enter-done {
+    z-index: 1001;
+    pointer-events: all;
+  }
+  &,
+  &.fade-enter,
+  &.fade-exit-active,
+  &.fade-exit-done {
+    opacity: 0;
+    transform: scale(0.9);
+  }
+  &.fade-enter-active,
+  &.fade-enter-done {
+    opacity: 1;
+    transform: scale(1);
+  }
+
+  ${props => props.theme.media.medium`
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    height: auto;
+    max-height: 90vh;
+    max-width: ${props.width || `${props.theme.sizes.large}px`}
+    width: ${props.width || 'auto'};
+    border-radius: ${props.theme.borderRadiusUnit}px;
+
+    &,
+    &.fade-enter,
+    &.fade-exit-active,
+    &.fade-exit-done {
+      transform: scale(0.9) translateX(-50%) translateY(-50%);
+    }
+    &.fade-enter-active,
+    &.fade-enter-done {
+      opacity: 1;
+      transform: scale(1) translateX(-50%) translateY(-50%);
+    }
+  `}
+  @media screen and (prefers-reduced-motion: reduce) {
+    transition: none;
+  }
+`;
+
+const RenderMoreFilters: FunctionComponent<RenderMoreFiltersProps> = ({
```

Comment:
Could we call this by it's component name, which I would argue is `MoreFilters`?

nit: I would also call this component `MoreFiltersModal` as that's how we would say it.